erpnext/quality_management/report/review/review.json

```diff
@@ -1,21 +1,22 @@
 {
- "add_total_row": 0, 
- "creation": "2018-10-16 12:28:43.651915", 
- "disabled": 0, 
- "docstatus": 0, 
- "doctype": "Report", 
- "idx": 0, 
- "is_standard": "Yes", 
- "modified": "2018-10-16 15:23:25.667237", 
- "modified_by": "Administrator", 
- "module": "Quality Management", 
- "name": "Review", 
- "owner": "Administrator", 
- "prepared_report": 0, 
- "query": "SELECT\n  `tabQuality Action`.name as \"Name:Data:200\",\n  `tabQuality Action`.action as \"Action:Select/[corrective,Preventive]:200\",\n  `tabQuality Action`.review as \"Review:Link/Quality Review:200\",\n  `tabQuality Action`.date as \"Date:Date:120\",\n  `tabQuality Action`.status as \"Status:Select/Planned:150\"\nFROM\n  `tabQuality Action`\nWHERE\n  `tabQuality Action`.type='Quality Review'\n  \n  ", 
- "ref_doctype": "Quality Action", 
- "report_name": "Review", 
- "report_type": "Query Report", 
+ "add_total_row": 0,
+ "creation": "2018-10-16 12:28:43.651915",
+ "disable_prepared_report": 0,
+ "disabled": 0,
+ "docstatus": 0,
+ "doctype": "Report",
+ "idx": 0,
+ "is_standard": "Yes",
+ "modified": "2020-01-26 20:14:45.354719",
+ "modified_by": "Administrator",
+ "module": "Quality Management",
+ "name": "Review",
+ "owner": "Administrator",
+ "prepared_report": 0,
+ "query": "SELECT\n  `tabQuality Action`.name as \"Name:Data:200\",\n  `tabQuality Action`.corrective_preventive as \"Action:Select/[Corrective,Preventive]:200\",\n  `tabQuality Action`.document_type as \"Document Type:Select/[Quality Review, Quality Feedback]:200\",\n  `tabQuality Action`.date as \"Date:Date:120\",\n  `tabQuality Action`.status as \"Status:Select/Planned:150\"\nFROM\n  `tabQuality Action`\nWHERE\n  `tabQuality Action`.document_type='Quality Review'\n  \n  ",
+ "ref_doctype": "Quality Action",
+ "report_name": "Review",
+ "report_type": "Query Report",
```

Comment:
Please add permission for "Quality Manager" by default

Reply:
Added Quality Manager permission